docs/design/autoware-architecture/.pages

```diff
@@ -0,0 +1,10 @@
+nav:
+  - index.md
+  - Control
+  - Localization
+  - Perception
+  - Planning
+  - Sensing
+  - Vehicle
+  - Map
```

Comment:
We use `dash-case`. So you should use lower-case.
https://developers.google.com/style/filenames?hl=en

```suggestion
  - control
  - localization
  - perception
  - planning
  - sensing
  - vehicle
  - map
```

Reply:
fixed in 81d4bdcde01a5

Reply:
Thank you, but you should also update the directory names.
![image](https://user-images.githubusercontent.com/31987104/167340558-bfc1834a-4d03-4031-9c95-feec39ac0e98.png)

Reply:
Fixed in [f979b52](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/f979b52790967555d56a4a850035c0074586287e)